.github/CONTRIBUTING.md

```diff
@@ -4,87 +4,92 @@ Share your knowledge with the community by contributing to Magento DevDocs!
+   In general, you should use `master` as the base branch when creating a PR. If your contribution is related to a release that is in progress, use a version-specific integration branch, such as `develop`.
 
-Provide links to devdocs for the files you are changing. This is not required if the list is long.
+1. Complete the pull request template.
 
-If you are updating an example from source code, include a link to the file in the repository.
+   We will close your pull request if you fail to complete the template.
 
-The DevDocs team and Maintainers will review the PR and help with formatting and navigation.
+1. After creating a pull request, a DevDocs staff member or maintainer will review it and may ask you to make revisions.
 
-**Note:** If you have not signed the [Magento Contributor Agreement](https://magento.com/content/magento-contributor-agreement), the PR provides a link. We require a signed form and agreement to the terms for contribution.
-
-**Tip!** If you are not sure where to start contributing, check out our GitHub issues labeled [help wanted](https://github.com/magento/devdocs/issues?q=is%3Aissue+is%3Aopen+label%3A%22help+wanted%22) and [good first issue](https://github.com/magento/devdocs/issues?q=is%3Aissue+is%3Aopen+label%3A%22good+first+issue%22).
+**Note:** If you have not signed the [Magento Contributor Agreement](https://magento.com/content/magento-contributor-agreement), the pull request provides a link. Magento requires a signed form and agreement to the terms for contribution.
 
 ## Contribution guidelines
 
-Write content using [kramdown](https://kramdown.gettalong.org/), which is a simple markup language. We use kramdown, Liquid, and [Jekyll](https://jekyllrb.com/) to generate a static site hosted through [GH Pages](https://help.github.com/articles/what-is-github-pages/). Check [Templates](#templates) for examples of styles and markdown.
-
-You can update existing or add new topics in their respective Magento 2 versioned directories (2.1, 2.2, 2.3, and onward). If you need help finding a directory for your content, we can help in your PR.
-
 The following guidelines may answer most of your questions and help you get started:
 
-1. Check [existing pull requests](https://github.com/magento/devdocs/pulls) and make sure you are not duplicating work!
-1. For large contributions or changes that include multiple files, [open an issue](#report-an-issue) and discuss it with us first. This may further prevent duplicate or unnecessary effort.
-1. [Chunk many small/medium changes](#tips-for-writing-content) into one or two PRs. This helps us to efficiently and effectively facilitate your contribution.
-1. Familiarize yourself with the existing documentation. Look through and search the guides to decide where to add your topics.
+- Write content using markdown. See the [Templates](#templates) section for examples.
```

Comment:
```suggestion
- Write content using Markdown. See the [Templates](#templates) section for examples.
```

Reply:
Our templates do not comply with pure Markdown markup. It makes sense to add information that we use [kramdown](https://kramdown.gettalong.org/) that implement several Markdown extensions.

Reply:
The _Templates_ section contains links to [Markdown](https://github.com/magento/devdocs/blob/master/guides/v2.1/contributor-guide/templates/basic_template.md) files. I'm not sure that this is a good idea. For example, if I'll follow the provided guidelines how to add notes,
```
{%raw%}
{%
include note.html
type='info'
content='This is a note callout. You can use these to provide important information on a topic.'
%}
{%endraw%}
```

I won't see the expected note.